OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/base/BaseNfcActivity.java

```diff
@@ -57,6 +57,9 @@
     public static final int REQUEST_CODE_PASSPHRASE = 1;
 
     protected Passphrase mPin;
+    protected Boolean mPw1ValidForMultipleSignatures;
+    protected Boolean mPw1ValidatedForSignature;
+    protected Boolean mPw1ValidatedForDecrypt; // Mode 82 does other things; consider renaming?
```

Comment:
Changed the Booleans to booleans. Also realized that I neglected to commit a change to the nfcDecryptSessionKey method, so that is added as well.